lib/compat/wordpress-6.3/navigation-block-preloading.php

```diff
@@ -0,0 +1,56 @@
+<?php
+/**
+ * Patches resources loaded by the block editor page
+ * to include Navigation posts.
+ *
+ * @package gutenberg
+ */
+
+/**
+ * Preloads requests needed for Navigation posts
+ *
+ * @param array $preload_paths    Preload paths to be filtered.
+ * @return array
+ */
+function gutenberg_preload_navigation_permissions( $preload_paths ) {
```

Comment:
Likely I am missing something, why is this named .."permissions"?

Reply:
Cos I copy/pasted 🤦